docs/guides/guides/module-api.mdx

```diff
@@ -56,31 +56,31 @@ node e2e-run-tests.js
-| `record`                  | _boolean_              | Whether to record the test run                                                                                                                                                                           |
-| `reporter`                | _string_               | Specify a [Mocha reporter](/guides/tooling/reporters)                                                                                                                                                    |
-| `reporterOptions`         | _object_               | Specify [Mocha reporter](/guides/tooling/reporters) options                                                                                                                                              |
-| `runnerUi`                | _boolean_              | Whether to display the Cypress Runner UI. Defaults to `false` when [Test Replay](/guides/cloud/test-replay) is enabled. Otherwise defaults to `true`.                                                    |
-| `slowTestThreshold`       | _number_               | Time, in milliseconds, to consider a test "slow" during `cypress run`. A slow test will display in orange text in the default reporter.                                                                  |
-| `spec`                    | _string_               | Specify the specs to run, see examples below                                                                                                                                                             |
-| `tag`                     | _string_               | Identify a run with a tag or tags                                                                                                                                                                        |
-| `testingType`             | _string_               | Specify the type of tests to execute; either `e2e` or `component`. Defaults to `e2e`                                                                                                                     |
+| Option                    | Type                  | Description                                                                                                                                                                                              |
+| ------------------------- | --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `autoCancelAfterFailures` | _number_ \| _boolean_ | Specify the number of failures to cancel a run being recorded to the Cloud or `false` to disable auto-cancellation.                                                                                      |
```

Comment:
```suggestion
| `autoCancelAfterFailures` | _number_ \| _false_ | Specify the number of failures to cancel a run being recorded to the Cloud or `false` to disable auto-cancellation.                                                                                      |
```

Reply:
False is the correct option here if a number is not specified

Reply:
Thanks for the correction. I was unfamiliar with the concept of TypeScript literal types.
I will revert to using `_false_` as a Type, so the only correction in this PR will be the rendering of the pipe symbol.